templates/repo/issue/labels/labels_selector_field.tmpl

```diff
@@ -0,0 +1,45 @@
+<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-label dropdown">
+	<a class="text gt-df gt-ac muted">
+		<strong>{{.locale.Tr "repo.issues.new.labels"}}</strong>
+		{{if and .HasIssuesOrPullsWritePermission (not .Repository.IsArchived)}}
+			{{svg "octicon-gear" 16 "gt-ml-2"}}
+		{{end}}
+	</a>
+	<div class="filter menu" {{if .Issue}}data-action="update" data-issue-id="{{$.Issue.ID}}" data-update-url="{{$.RepoLink}}/issues/labels"{{else}}data-id="#label_ids"{{end}}>
+		<div class="header" style="text-transform: none;font-size:16px;">{{.locale.Tr "repo.issues.new.add_labels_title"}}</div>
+		{{if or .Labels .OrgLabels}}
+			<div class="ui icon search input">
+				<i class="icon gt-df gt-ac gt-jc">{{svg "octicon-search" 16}}</i>
+				<input type="text" placeholder="{{.locale.Tr "repo.issues.filter_labels"}}">
+			</div>
+		{{end}}
+		<a class="no-select item" href="#">{{.locale.Tr "repo.issues.new.clear_labels"}}</a>
+		{{if or .Labels .OrgLabels}}
+			{{$previousScope := "_no_scope"}}
+			{{range .Labels}}
+				{{$scope := .Scope}}
+				{{$exclusiveScope := .ExclusiveScope}}
+				{{if and (ne $previousScope "_no_scope") (ne $previousScope $scope)}}
+					<div class="ui divider"></div>
+				{{end}}
+				{{$previousScope = $scope}}
```

Comment:
I think the precondition mentioned in https://github.com/go-gitea/gitea/pull/22585#discussion_r1110452099 doesn't apply anymore with this change, right?
So it should be possible to use `scope := ""` now?

Reply:
`"_no_scope"` still prevents adding an extra divider at the start of the list, and between repository and organization labels.

Reply:
I am not quite sure, but wouldn't `""` produce the same result?

Reply:
Yup, I still think current approach is good enough. That's the simplest one, unless introducing other variables or use nil, which makes the if more complex.

Reply:
It's not the same result. With `""` it would add a divider at the start of the list depending if the first label has a scope, which I don't think is right.

Reply:
> I am not quite sure, but wouldn't `""` produce the same result?

The `scope` could be empty, it's difficult to distinguish an empty scope or "no scope"